libraries/SITL/SIM_Frame.h

```diff
@@ -44,19 +45,92 @@ class Frame {
     static Frame *find_frame(const char *name);
     
     // initialise frame
-    void init(float mass, float hover_throttle, float terminal_velocity, float terminal_rotation_rate);
-
+    void init(const char *frame_str);
+    
```

Comment:
got a little extra tab here, no big deal of course..

Reply:
fixed